pyproject.toml

```
@@ -23,7 +23,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.7.0"
-deepdiff = "^5.5.0"
+deepdiff = ">=5.5.0"
```

```suggestion
deepdiff = ">=5.5.0 <7.0"
```
And we should be fine. I've checked all unittests are passing with 6.3.